core/client/app/templates/settings/connections/edit.hbs

```diff
@@ -0,0 +1,18 @@
+<section class="view-content">
+
+{{gh-edit-client
+    isEditing=isEditing
+    trustedDomains=model.trusted_domains
```

Comment:
I'm going to backtrack on my suggestion to pass each value into the component. Change this to pass a model and internally use the passed in model.